index.html

```diff
@@ -0,0 +1,125 @@
+            <li><a href=""> Logout </a></li>
+        </ul>
+    </div>
+    <nav class="nav_bar padding-x">
+        <a href="index.html"><img class="my-logo" src="./imgs/icons/logo.png" alt="My Log"></a>        
+        <img class="humburger" src="./imgs/icons/humburger.jpg" alt="Humburger Icon">
+        <div class="close-btn">
+            <p>x</p>
+        </div>
+        <ul>
+            <li><a class="about" href="index.html" >Home</a></li>
+            <li><a class="" href="about.html" >About</a></li>
+            <li><a href="" >Programs</a></li>
+            <li><a href="" >Join</a></li>
+            <li><a href="" >Sponser</a></li>
+            <li><a href="" >News</a></li>
+            <li  class="cc" ><ahref="" >CC campign</a></li>
+        </ul>
+    </nav>
+    <section class="home-section padding-x">
+        <h3>"Hello! Sharing World"</h3>
+        <h1>CREATIVE COMMONS GLOBAL SUMMIT 2015</h1>
+        <div>
+            <p>
+                A joy full celebration beleving in the value of opennes and Sharing
+                , Creating a posetive change with people from over 80 countries is taking
+                place in Ocutober. In Korea
+            </p>
+        </div>
+        <p class="lg-text">2015.10.15(THU) ~ 16(FRI)</p>
+        <p>@ National Museum of Korea, Art Center Nabi and More</p>
+    </section>
+    <section class="main-program padding-x bg-black">
+        <h1>Main Programs</h1>
+        <span class="red_underline"></span>
+        <div class="main-program-content">
+            <div>
+                <img src="./imgs/icons/feature-image-4.svg" alt="Lecture">
+                <h2>Lecture</h2>
+                <p>Listen to the speaker from varous country about the message of sharing and opening</p>
+            </div>
+            <div>
+                <img src="./imgs/icons/Multiple File Types.svg" alt="CC Exheption">
+                <h2>CC Exehption</h2>
+                <p>Listen to the speaker from varous country about the message of sharing and opening</p>
+            </div>
+            <div>
+                <img src="./imgs/icons/logo_bagde.svg" alt="Forum">
+                <h2>Forum</h2>
+                <p>Listen to the speaker from varous country about the message of sharing and opening</p>
+            </div>
+            <div>
+                <img src="./imgs/icons/Editable Vectors.svg" alt="Workshop">
+                <h2>Workshop</h2>
+                <p>Listen to the speaker from varous country about the message of sharing and opening</p>
+            </div>
+            <div>
+                <img src="./imgs/icons/Monthly updates.svg" alt="CC ignite">
+                <h2>CC Ignite</h2>
+                <p>Listen to the speaker from varous country about the message of sharing and opening</p>
+            </div>
+        </div>
+        <h1 class="h1-desk">SEE THE WHOLE PROGRAMS</h1>
+        <h1 class="h1-mob">Join CC Global summit 2015</h1>
+    </section>
```

Comment:
Kindly update the main program section to match the behance design better on desktop🙏

Your design:
<img width="508" alt="image" src="https://user-images.githubusercontent.com/37116322/234413694-1eb512a0-4c73-4b92-9ca1-a73b9a3b05c6.png">

Behance design:
<img width="628" alt="image" src="https://user-images.githubusercontent.com/37116322/234413742-b9f2c35d-3e8d-43d2-a624-61e9c1394ca1.png">

A few things to note:
- Please make the images smaller so it matches the design better
- Currently, it's hard to see the background of each of the cards. Please make it more visible
- Please add more bottom padding to the section

Reply:
# 🎁🎁🎁 Thanks a lot, I will try my best to correct it. 🎀🎀🎀